src/lcgs/parse.rs

```diff
@@ -69,7 +69,14 @@ fn non_0_digit() -> Parser<'static, u8, u8> {
 
 /// Parser that parses 0 or more whitespace characters discards them. Include newlines and tabs.
 fn ws() -> Parser<'static, u8, ()> {
-    one_of(b" \t\r\n").repeat(0..).discard()
+    (one_of(b" \t\r\n").discard() | comment())
+        .repeat(0..)
+        .discard()
+}
+
+/// Parser for comments that matches on '//' and whatever until \r or \n, and discards everything
+fn comment() -> Parser<'static, u8, ()> {
+    (seq(b"//") + none_of(b"\r\n").repeat(0..) + one_of(b"\r\n")).discard()
 }
```

Comment:
For at det er muligt at lave inline kommentar, så skal alle parsere så også implementere ws() i slutningen af deres parser decl.
eg.
`const dinmor = 2; //Hej mit navn er åge`

Reply:
Her er en test der viser det :)
```rust
    #[test]
    fn test_comment_05() {
        let input = br#"const max_health = 1; // hunter2
        const vvvv = 1"#;
        let parser = root();
        assert!(parser.parse(input).is_ok());
        assert_eq!(parser.parse(input).unwrap().decls.len(), 2);
    }
```

Reply:
_huh, nvm_